content/en/docs/project-user-guide/application-workloads/daemonsets.md

```diff
@@ -21,13 +21,14 @@ DaemonSets are very helpful in cases where you want to deploy ongoing background
 ## Prerequisites
 
 - You need to create a workspace, a project and an account (`project-regular`). Please refer to [Create Workspace, Project, Account and Role](../../../quick-start/create-workspace-and-project) if they are not ready yet.
-- You need to sign in with the `project-admin` account and invite `project-regular` to the corresponding project. Please refer to [these steps to invite a member](../../../quick-start/create-workspace-and-project#task-3-create-a-project).
+
+- You need to sign in with the `project-admin` account and invite `project-regular` to the corresponding project. Please refer to [the steps to invite a member](../../../quick-start/create-workspace-and-project#task-3-create-a-project).
 
 ## Create a DaemonSet
 
 ### Step 1: Open Dashboard
 
-Log in the console as `project-regular`. Go to **Workloads** of a project, choose **DaemonSets** and click **Create**.
+Log in the console as `project-regular`. Go to **Application Workloads** of a project, choose **DaemonSets** and click **Create**.
```

Comment:
Users cannot choose **DaemonSets** directly if they go to **Application Workloads**. The tab is in **Workloads**. If you want to add **Application Workloads** here, try this:

Go to **Application Workloads** of a project, select **Workloads**,  and click **Create** under the tab **DaemonSets**.

Reply:
I see, I will update it in a different PR.